### PR TITLE
Add cluster redeploy and update faulty cluster member restore instructions

### LINKED ISSUE #1702: Update troubleshooting re: cluster issues
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior

In our (Sensu's) troubleshooting efforts with multiple customers, we've run into the need to describe how to troubleshoot cluster issues (e.g., stopping cluster members and removing `/var/lib/sensu/etcd` and restarting). This should be present in the troubleshooting doc.

## Current Behavior

Currently, this isn't present and is a fairly common issue that folks run across. 

## Scoping

* Needs to ensure that we're following recommended best practices for etcd cluster troublshooting
* Ensure that we address that this is a destructive action
* Could also include the new `dump` feature and discuss reimporting data, though this may be a separate issue to track/write

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- In Troubleshoot Sensu, adds a section of instructions for completely removing a cluster and redeploying from scratch.
- In Run a Sensu cluster, adds the reason to remove a faulty cluster member, cleans up instructions for replacing a faulty cluster member, adds a note to emphasize "existing" rather than "new" in config for the new member, and adds a link to the redeploy cluster in troubleshooting.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/1702

### CODE REVIEW

File: content/sensu-go/6.2/operations/deploy-sensu/cluster-sensu.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ For the new node `backend-4` with IP address `10.0.0.4`:
 etcd-advertise-client-urls: "http://10.0.0.4:2379"
 etcd-listen-client-urls: "http://10.0.0.4:2379"
 etcd-listen-peer-urls: "http://0.0.0.0:2380"
-etcd-initial-cluster: "backend-1=http://10.0.0.1:2380,backend-2=http://10.0.0.2:2380,backend-3=http://10.0.0.3:2380,backend-4=https://10.0.0.4:2380"
+etcd-initial-cluster: "backend-1=http://10.0.0.1:2380,backend-2=http://10.0.0.2:2380,backend-3=http://10.0.0.3:2380,backend-4=http://10.0.0.4:2380"
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/operations/deploy-sensu/cluster-sensu.md:185]: I noticed that some of our code examples seem to mix http and https...I think http is correct at this point (later instructions discuss securing cluster members) but please correct me if not.

File: content/sensu-go/6.2/operations/deploy-sensu/cluster-sensu.md [hunk 2]
```diff
@@ -234,10 +236,10 @@ You will receive a sensuctl response that lists all cluster members:
...
        ID            Name             Peer URLs                Client URLs
 ────────────────── ─────────── ───────────────────────── ─────────────────────────
-a32e8f613b529ad4   backend-1    https://10.0.0.1:2380     https://10.0.0.1:2379  
-c3d9f4b8d0dd1ac9   backend-2    https://10.0.0.2:2380     https://10.0.0.2:2379
-c8f63ae435a5e6bf   backend-3    https://10.0.0.3:2380     https://10.0.0.3:2379
-2f7ae42c315f8c2d   backend-4    https://10.0.0.4:2380     https://10.0.0.4:2379
+a32e8f613b529ad4   backend-1    http://10.0.0.1:2380      http://10.0.0.1:2379  
+c3d9f4b8d0dd1ac9   backend-2    http://10.0.0.2:2380      http://10.0.0.2:2379
+c8f63ae435a5e6bf   backend-3    http://10.0.0.3:2380      http://10.0.0.3:2379
+2f7ae42c315f8c2d   backend-4    http://10.0.0.4:2380      http://10.0.0.4:2379
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/operations/deploy-sensu/cluster-sensu.md:242]: Again here, I think `http` is correct, but please confirm

File: content/sensu-go/6.2/operations/deploy-sensu/cluster-sensu.md [hunk 3]
```diff
@@ -375,6 +371,8 @@ sensu-backend start \
 
 ## Troubleshoot clusters
 
+If you want to restart a cluster, read [Remove and redeploy a cluster][23].
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/operations/deploy-sensu/cluster-sensu.md]: I want to provide an example or two of a reason to restart a cluster. Are there reasons to do this aside from more than one cluster member failing and taking the whole cluster down?

File: content/sensu-go/6.2/operations/maintain-sensu/troubleshoot.md [hunk 4]
```diff
@@ -677,6 +677,48 @@ https://backend02:2379, e98ad7a888d16bd6, 3.3.22, 1.0 MB, true, 144, 18619245
 https://backend03:2379, bc4e39432cbb36d, 3.3.22, 1.0 MB, false, 144, 18619245
 {{< /code >}}
 
+### Remove and redeploy a cluster
+
+In some cases, you may need to completely remove a cluster and redeploy it.
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/operations/maintain-sensu/troubleshoot.md]: Here again I want to expand on "In some cases..." with examples.

**Review by mgibson323 — APPROVED** [review]
This LGTM, with what I know of the process.

File: content/sensu-go/6.2/operations/deploy-sensu/cluster-sensu.md (hunk 3 above)
Comment (hillaryfraley) [reply content/sensu-go/6.2/operations/deploy-sensu/cluster-sensu.md]: Answered in slack: https://sensu.slack.com/archives/C60EEQFH8/p1620224379273200

**Pull request merged**